### ISSUE TITLE
Improvement request for "Goat Shrine" and "Cauldron"

### ISSUE THREAD

**Issue by Chance4us** [diasurgical/devilutionX#6286]
Hello my friends.
We already have the option to disable the "crippling shrines".
Unfortunately, this means that the random shrines mentioned above are completely inoperable.
As a result, the game loses variety.

The idea is to let the random shrines still work,
but to exclude the "crippling shrines" contained in the random calculation of the random shines.
What do you think about?

**Comment by Chance4us** [comment]
Of course, we could only disable shrines that cause permanent damage to the hero.
Shrines with short-term negative effects, for example "Murphy's Shrine" could be retained so that not only positive effects occur when using random shrines. It's just to avoid permanent damage to the hero.

**Comment by qndel** [comment]
Not letting players click and changing game mechanics are two different things
